tests/test-runner/bin/test-runner.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/python3
```

Comment:
> This commit changes test-runner.py to also use /usr/bin/python3,
> enabling the change during buildtime and fixing a minor lintian issue
> **on Debian systems, which only have python3 installed**.

The highlighted part of the commit message seems to suggest (all) Debian systems don't provide python2. I tested this change on a Debian8 (Python 2.7.9) but the number of lintian warning/errors regarding `test-runner.py` seems to be the same as the unpatched version:

```
root@linux:~# lsb_release -a
No LSB modules are available.
Distributor ID:	Debian
Description:	Debian GNU/Linux 8.0 (jessie)
Release:	8.0
Codename:	jessie
root@linux:~# python -V
Python 2.7.9
root@linux:~# grep runner /var/tmp/patched /var/tmp/vanilla 
/var/tmp/patched:E: zfs-test: python-script-but-no-python-dep usr/share/zfs/test-runner/bin/test-runner.py
/var/tmp/patched:E: zfs-test: python-script-but-no-python-dep usr/share/zfs/test-runner/bin/zts-report.py
/var/tmp/vanilla:E: zfs-test: python-script-but-no-python-dep usr/share/zfs/test-runner/bin/test-runner.py
/var/tmp/vanilla:E: zfs-test: python-script-but-no-python-dep usr/share/zfs/test-runner/bin/zts-report.py
root@linux:~# 
```

Reply:
Thanks for the review and for catching the wrong phrasing of the commit message!
I just pushed a reworded version.

Additionally tested by building zfs while explicitly configuring for python2, and adapting the appropriate fields in the debian/control file - The lintian errors don't appear, and the shebang of test-runner.py is set to `/usr/bin/python2`